src/numerics/include/metaphysicl/dualnumber.h

```diff
@@ -404,11 +430,62 @@ DualNumber_std_unary(sinh, std::cosh(in.value()),)
 DualNumber_std_unary(cosh, std::sinh(in.value()),)
 DualNumber_std_unary(tanh, sech_in * sech_in, T sech_in = 1 / std::cosh(in.value()))
 DualNumber_std_unary(abs, (in.value() > 0) - (in.value() < 0),) // std < and > return 0 or 1
-DualNumber_std_unary(fabs, (in.value() > 0) - (in.value() < 0),) // std < and > return 0 or 1
+DualNumber_equiv_unary(fabs, abs)
 DualNumber_std_unary(norm, 2. * in.value(),)
 DualNumber_std_unary(ceil, 0,)
 DualNumber_std_unary(floor, 0,)
 
+#if __cplusplus >= 201103L
+DualNumber_equiv_unary(llabs, abs)
+DualNumber_equiv_unary(imaxabs, abs)
+DualNumber_equivfl_unary(fabs)
+DualNumber_equivfl_unary(exp)
+DualNumber_std_unary(exp2, std::log(T(2))*std::exp(in.value()),)
+DualNumber_equivfl_unary(exp2)
+DualNumber_std_unary(expm1, std::exp(in.value()),)
```

Comment:
More than a tiny bit, but I was more worried about accuracy.  ... but I guess that shouldn't be a worry?  The main justification for expm1 vs exp-1 is to avoid growing truncation error by 1/funcval when funcval->0... but that wouldn't occur in the opposite direction.  We *would* grow truncation error in your case when funcval->-1, though.  Not sure whether that's a practical concern (if people are evaluating x << 0 then they won't be using expm1, right?), but I think I'd rather risk O(1) cost than O(unbounded) relative error.